python/core/symbology-ng/qgssymbollayerutils.sip

```diff
@@ -501,4 +501,20 @@ class QgsSymbolLayerUtils
       */
     static void mergeScaleDependencies( int mScaleMinDenom, int mScaleMaxDenom, QgsStringMap& props );
 
+    /**
+     * Encodes a reference to a parametric SVG into SLD, as a succession of parametric SVG using URL parameters,
+     * a fallback SVG without parameters, and a final fallback as a mark with the right colors and outline for systems
+     * that cannot do SVG at all
+     * @note added in 3.0
+     */
+    static void parametricSvgToSld( QDomDocument &doc, QDomElement &graphicElem,
```

Comment:
I'd suggest:

1. Rename color to fillColor
2. Make this return the QDomElement instead of passing it as an argument